src/test/scala/io/iohk/ethereum/vm/OpCodeFunSpec.scala

```diff
@@ -103,64 +120,48 @@ class OpCodeFunSpec extends FunSuite with OpCodeTesting with Matchers with Prope
     }
   }
 
-  ignore("ADDRESS") {
-    // to be implemented
-  }
-
-  ignore("BALANCE") {
```

Comment:
This suite won't pass if we remove that. See how `verifyAllOpCodesRegistered()` for details.